specification/dimensions/billingperiodstart.md

```diff
@@ -0,0 +1,32 @@
+# Billing Period Start
+
+Billing period represents the time window for which you have or will receive an invoice for. Billing period represents the time window for which you have or will receive an invoice for. The time window is inclusive of the start date and exclusive of the end date.
+
+Billing Period Start represents the beginning of a billing period.
+
+The BillingPeriodStart column MUST be present in the billing data. This column MUST conform to [FOCUS Date/Time Requirements](#date/timerequirements) and MUST NOT contain null values. The sum of the Billed Cost metric for line items in a given billing period MUST match the total cost of the invoices received for that billing period.
```

Comment:
Is the casing for "Billed Cost metric" correct? Should it be "billed cost metric"

Reply:
This is correct. If we're referring to a concept (e.g. billing account, resource), we can avoid capitalizing it. When we're referring to the actual column we use the display name.